README.md

```diff
@@ -0,0 +1,400 @@
+# MOA Command Line Tool
+
+This project contains the `moactl` command line tool that simplifies the use of Amazon Red Hat OpenShift, also known as _MOA_.
+
+# Quickstart guide
+
+This guide walks through setting up your first Amazon Red Hat OpenShift cluster using `moactl`.
+
+This includes the following steps:
+
+* Installation prerequisites
+* Initializing your AWS account
+* Creating your cluster
+* Accessing your cluster
+* Installing an addon to your cluster
+* Creating admin users to your cluster
+* Cleaning up
+
+By the end of this guide you will have a Amazon Red Hat OpenShift cluster running in your AWS account.
+
+## Installation prerequisites
+
+Complete the following prerequisites before creating your MOA cluster.
+
+### Select an AWS account to use
```

Comment:
@jeremyeder @jharrington22 @vkareh I added a new section here based on the customer cloud subscription doc. Appreciate feedback on how much of this info will be useful for the customer to know.

I think we still have not yet addressed the use case of a customer with an existing SCP that in some way limits the use of moa. I recommended not having an SCP at all.